zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterOption.java

```diff
@@ -28,7 +28,8 @@
   public static final transient String SCOPED = "scoped";
   public static final transient String ISOLATED = "isolated";
 
-  boolean remote;
+  // always set it as true, keep this field just for backward compatibility
+  boolean remote = true;
```

Comment:
why leave this variable? it won't be helpful for backward compatibility if it is removed in the constructor, and setRemote, isRemote anyway?

Reply:
Because this correspond to one field in `interpreter-setting.json`. And I don't want to break compatibility of `interpreter-setting.json` for now. I have a plan to define the spec of `interpreter-setting.json` & `interpreter.json`, at that time I may break the compatibility of them. But for now, I just want to keep them backward compatible.

Reply:
ok